core/prepare_api.php

```diff
@@ -104,3 +104,52 @@ function prepare_version_string( $p_project_id, $p_version_id ) {
 
 	return $t_version_text;
 }
+
+/**
+ * Prepares avatar for raw outputting (only avatar image).
+ *
+ * @param Avatar $p_avatar 			An instance of class Avatar.
+ * @param string $p_class_prefix	CSS class prefix to add to the avatar's surrounding div and to the img.
+ *   The CSS classes to implement will be named [$p_class_prefix]_[$p_size]_avatar_container and 
+ *   [$p_class_prefix]_[$p_size]_avatar.
+ * @param integer $p_size    		Image maximum size.
+ * @return the HTML string of the avatar.
+ */
+function prepare_raw_avatar( $p_avatar, $p_class_prefix, $p_size) {
```

Comment:
Please revise spacing inside () across all changes.  For example, the function definition and calls just look as follows:

```
func1( $p_param1, $p_param2 )
```

Notice the spaces inside the ().